### ISSUE TITLE
Sometimes auto-expand fails with error

### ISSUE THREAD

**Issue by charlesivie** [aws/graph-explorer#146]
**Community Note**
* Please use a 👍 reaction to provide a +1/vote. This helps the community and maintainers prioritize this request.
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.

**Describe the bug**
Sometimes, double clicking to expand a node shows an error:

```
useEntities.ts:106 Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Transition2 which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here: https://reactjs.org/link/strict-mode-find-node
    at div
    at Transition2 (http://localhost:5173/explorer/node_modules/.vite/deps/react-transition-group.js?v=7c744aef:109:30)
    at CSSTransition2 (http://localhost:5173/explorer/node_modules/.vite/deps/react-transition-group.js?v=7c744aef:528:35)
    at div
    at TransitionGroup2 (http://localhost:5173/explorer/node_modules/.vite/deps/react-transition-group.js?v=7c744aef:902:30)
    at div
    at div
    at NotificationProvider (http://localhost:5173/explorer/src/components/NotificationProvider/NotificationProvider.tsx:71:3)
    at div
    at ThemeProvider (http://localhost:5173/explorer/src/core/ThemeProvider/ThemeProvider.tsx:37:3)
    at http://localhost:5173/explorer/node_modules/.vite/deps/react-dnd.js?v=7c744aef:1639:23
    at QueryClientProvider2 (http://localhost:5173/explorer/node_modules/.vite/deps/react-query.js?v=7c744aef:2647:21)
    at div
    at ConnectedProvider (http://localhost:5173/explorer/src/core/ConnectedProvider/ConnectedProvider.tsx:37:5)
    at Router (http://localhost:5173/explorer/node_modules/.vite/deps/react-router-dom.js?v=7c744aef:2744:15)
    at HashRouter (http://localhost:5173/explorer/node_modules/.vite/deps/react-router-dom.js?v=7c744aef:3158:5)
```

 - OS: [e.g. Amazon Linux 2]
 - Browser: [e.g. Google Chrome]
 - Graph Explorer Version: [e.g. 1.2.1]

**Comment by AndreaNassisi** [comment]
Can we replicate it?

**Comment by xiazcy** [comment]
Issue resolved in #211.
